devel/soprano/Portfile

```diff
@@ -42,7 +42,7 @@ configure.args-append \
                     -DSOPRANO_DISABLE_CLUCENE_INDEX=1
 
 pre-configure {
-    if {![file isfile "/System/Library/Frameworks/JavaVM.framework/Headers/jni.h"]} {
+        if {![file isfile "/System/Library/Frameworks/JavaVM.framework/Headers/jni.h"] && ![file isfile "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/JavaVM.framework/Versions/A/Headers/jni.h"]} {
```

Comment:
Did you intentionally change the indentation?

Reply:
No

Reply:
Cool. I'd fix that then. Any ideas who else might want to review before we do a merge?

Reply:
OK, I amended the commit. I don't know of anyone that should review this other than the maintainer. Maybe if there is someone who specializes in how macports works with XCode and Command-line tools? In any case, this fixes a known breakage, and so I would err on the side of committing it soon.